src/views/Request.vue

```diff
@@ -5,18 +5,22 @@
 				<h1>{{ t('libresign', 'Request Signatures') }}</h1>
 				<p>{{ t('libresign', 'Choose the file to request signatures.') }}</p>
 			</header>
-			<div class="content-request">
+			<div class="container-buttons" v-if="loading">
+				<NcLoadingIcon :size="64" name="Loading" />
+				<p>{{ t('libresign', 'Upload file') }}</p>
+			</div>
+			<div class="container-buttons" v-if="!loading">
 				<File v-show="!isEmptyFile"
 					:file="file"
 					status="0"
 					status-text="none"
 					@sidebar="showSidebar = true" />
-				<NcButton @click="showModalUploadFromUrl">
-					{{ t('libresign', 'Upload from URL') }}
-					<template #icon>
-						<LinkIcon :size="20" />
-					</template>
-				</NcButton>
+				<!-- <NcButton @click="showModalUploadFromUrl"> -->
```

Comment:
Maybe would be best to hide this item:

```suggestion
				<NcButton @click="showModalUploadFromUrl" v-if="false">
```

Will be less changes and the linter don't will cry